### ISSUE TITLE
Question about the flow estimation coordinates in Waymo

### ISSUE THREAD

**Issue by Robertwyq** [Lilac-Lee/FastNSF#5]
Thanks for your great work.

I notice that the flow estimation in Waymo is under the ego coordinates. 
pc1 and pc2 is under the ego coordinates, and adjust the gt flow by transformation.

Even though the evaluation is correct, the scene flow estimated by the model is a mixture of (scene motion-ego motion). Is there any problem in estimating the motion of a relatively static or completely static point? 

Perhaps I have some misunderstandings, I am looking forward to your reply.

**Comment by Lilac-Lee** [comment]
Hi @Robertwyq, scene flow is defined as the motions in the 3D, including both rigid motions (AV motion) and dynamic motions (relative to AV).

Cheers.

**Comment by Robertwyq** [comment]
Yes I understand, but it seems the model is estimated the dynamic motions(relative to AV)?

**Comment by Lilac-Lee** [comment]
Hi @Robertwyq, I did not fully understand your question. By definition, scene flow includes both rigid and dynamic motions. Therefore, the model needs to estimate both the rigid and dynamic motions. 

Cheers.
